### ISSUE TITLE
[BUG] Drawer modal height changes when focus switches between input fields on mobile device

### ISSUE THREAD

**Issue by KemalPajevic** [kirbydesign/designsystem#3730]
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
If you have a drawer modal with multiple input fields, switching between the fields changes the height of the drawer on a mobile device (tested on iOS and Android).

In an Capacitor iOS app, it seems to loose the additional height it needs to offset the keyboard.
In a web page opened in iOS Safari or Android Chrome the behavior seems more erratic (se the screen recording below).

## Describe how to reproduce the bug

1. Add a component with multiple input fields.
2. Open a drawer with that component.
3. Switch focus between the input fields.
4. See that the drawer height changes.

A demo of the issue can be seen on this [Stackblitz](https://stackblitz.com/edit/stackblitz-starters-uts7j6) (open it in a mobile browser).

## Which Kirby version was used?
9.7.2 and 10.0.0.

## What was the expected behavior?

The height remains the same.

## Add any screenshots
Capacitor app in iOS:

https://github.com/user-attachments/assets/f8f8d5fb-e6f4-4053-8261-bc8e6f5707b8

In a web page in iOS Safari:

https://github.com/user-attachments/assets/f1dfc7eb-c1e5-44bc-8385-191b9746b78d

## Please complete the following information:
- OS: iOS, Android
- Browser: Safari, Chrome

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
